include/SFML/Graphics/Glsl.inl

```diff
@@ -0,0 +1,203 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2015 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+
+////////////////////////////////////////////////////////////
+/// \brief Helper function to copy sf::Transform to sf::Glsl::Mat3
+///
+////////////////////////////////////////////////////////////
+void initializeMatrix(Matrix<3, 3>& matrix, const Transform& transform);
```

Comment:
I think the name of these functions can be slightly misleading. Matrices are already initialized when they are constructed, so having to call these functions isn't necessary in order to use them even though that might be what people think. What these functions really do is already stated in their brief description: they copy data into the matrix. Less ambiguous names for these functions would probably be `copyMatrix` or `setMatrix`.